### ISSUE TITLE
Exif support?

### ISSUE THREAD

**Issue by killerbees19** [nanostudio-org/nano_photos_provider2#28]
Feature request: **Exif data support!** 🤓 (_exifModel, exifIso, exifFStop, exifExposure, …_)

As far as I can see this is currently not supported for data providers, so it would be required to patch…

* repository **nanogallery2**<br />file `jquery.nanogallery2.data_nano_photos_provider2.js`<br />at `jQuery.nanogallery2.data_nano_photos_provider2.JsonParseData()`

* repository **nano_photos_provider2**<br />file `nano_photos_provider2.json.class.php`<br />at `galleryJSON::PrepareData()`

I've already tested this with static dummy data, seems to work. I'm completely new to this great project, please forgive me if I'm missing an important part.

tl;dr: Would you love to see such a feature? Is someone working on something similar?

**Comment by JepKoha** [comment]
![nano-provider-exif-support](https://user-images.githubusercontent.com/44463306/99148023-ba1e8400-2685-11eb-8a4d-48ae102b73bd.jpg)
As I needed that feature for my use, I patched my own nano_photos_provider2.json.class.php v1.2.0 and jquery.nanogallery2.js v2.4.2 for exif data support...
You may find updated code parts for the PHP class and JS file in attached txt file
[nanogallery_photos_provider2-exif-data-support.txt](https://github.com/nanostudio-org/nano_photos_provider2/files/5540781/nanogallery_photos_provider2-exif-data-support.txt)
The idea is to fetch exif data in the file, populate a data array in the nano photo provider. Then array content populates exif data to be shown in the NanoGallery when informations are requested.
Hope this help.
Could certainly be improved, but works fine for me to display some exif data from photo files in the info popUp of a nanogallery. When upgraded, shows the expected exif data in NanoGallery when available.
Enjoy !
(By the way, to all contributors : Great App !-)
